backend/controllers/plan.ts

```diff
@@ -0,0 +1,40 @@
+import PlanService from '../service/plan'
+import PlanMembership from '../service/planMembership'
+import { type NextFunction } from 'express'
+
+export const createPlan = async (req: any, res: any, next: NextFunction): Promise<void> => {
+  try {
+    const {
+      description,
+      startDate,
+      endDate,
+      eventType
+    } = req.body
+
+    const planData = {
+      description,
+      startDate: new Date(startDate),
+      endDate: new Date(endDate),
+      specialEventType: eventType
+    }
+    const planRecord = await PlanService.initiatePlan(planData)
+
+    if (planRecord === undefined) throw new Error('initiate plan failed')
```

Comment:
placeholder error handlers

Reply:
do you have to check for null here as well or is the expected return `undefined`?

Reply:
For the prisma create methods, they either return the data or undefined.